### PR TITLE
fix: update skip / SEO

### PR DESCRIPTION
**Pull request by chalabi2**
## 1. Summary
Fixes the unwinder for sending rewards to osmosis or their parent chains

Adds better SEO Meta tags & banner images
## 2.Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] Feature
## 3. Implementation details

Upgreaded the skip api to v5 and changed the way user addresses are mapped to the skip execution hook

## Summary by CodeRabbit

- **New Features**
  - Introduced improved SEO and social media sharing metadata across multiple pages, including descriptions, keywords, author information, and Open Graph/Twitter card tags.
  - Added structured data in JSON-LD format for better search engine visibility and rich snippets.
  - Enhanced `RewardsModal` component with improved address handling, error management, and asset selection features.

- **Bug Fixes**
  - Improved address ordering and error handling in the `RewardsModal` component to prevent execution issues with undefined addresses.

- **Enhancements**
  - Updated `useSkipExecute` function to expect an array of `UserAddress` objects, improving type safety and consistency.
  - Expanded supported chains in the `WalletButton` component.

### CODE REVIEW

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>8408dbb</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/Dua8uM1fjPR8UJUDhcJzqZRoTXfB'>https://vercel.com/quicksilver/quicksilver/Dua8uM1fjPR8UJUDhcJzqZRoTXfB</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9444498227)

**Comment by chalabi2** [comment]
@joe-bowman I added better head tags for SEO alongside the skip api fixes. Can you please review them for each of the pages, (staking, assets, defi, governance) it is at the top of each page in between the <Head> tags. Things to review would include descriptions and such.

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>3d69a68</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/62NLRAgvHDFySiegPwkRLsJgGK3t'>https://vercel.com/quicksilver/quicksilver/62NLRAgvHDFySiegPwkRLsJgGK3t</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9444939612)

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>3da57c1</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/6mtLTy5XzAvDQQcZAkYrDfNDeodq'>https://vercel.com/quicksilver/quicksilver/6mtLTy5XzAvDQQcZAkYrDfNDeodq</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9445434587)

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>86467e5</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/22r1hs8B1gqPM52jdppjiwLD6Lyg'>https://vercel.com/quicksilver/quicksilver/22r1hs8B1gqPM52jdppjiwLD6Lyg</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9453761029)

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>e1ac4f5</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/2xxetjrnKbZPhxyM6w7JZwq8rUBS'>https://vercel.com/quicksilver/quicksilver/2xxetjrnKbZPhxyM6w7JZwq8rUBS</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9453907535)

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>7437449</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/6mGTh6ioSuYZsWbqNzV5NqAmiUac'>https://vercel.com/quicksilver/quicksilver/6mGTh6ioSuYZsWbqNzV5NqAmiUac</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9454015502)

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>1ab7ac5</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/Gm8NAGj1DQznXH7BCnDR6CtLH3gn'>https://vercel.com/quicksilver/quicksilver/Gm8NAGj1DQznXH7BCnDR6CtLH3gn</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9505417954)

File: web-ui/pages/about.tsx [hunk 1]
```diff
@@ -17,38 +17,35 @@ const AboutPage = () => {
...
+          />
           <meta name="author" content="Quicksilver Zone" />
           <link rel="icon" href="/img/favicon-main.png" />
 
           <meta property="og:title" content="About - Quicksilver Zone" />
-          <meta
-            property="og:description"
-            content="Interchain liquid staking hub. Secure your stake with the user-focused liquid staking."
-          />
+          <meta property="og:description" content="STAKING SIMPLIFIED | LQUIDITY AMPLIFIED" />
```
Comment (joe-bowman) [inline web-ui/pages/about.tsx:29]: Typo

File: web-ui/pages/about.tsx [hunk 2]
```diff
@@ -17,38 +17,35 @@ const AboutPage = () => {
...
 
           <meta name="twitter:card" content="summary_large_image" />
           <meta name="twitter:title" content="About - Quicksilver Zone" />
-          <meta
-            name="twitter:description"
-            content="Interchain liquid staking hub. Secure your stake with the user-focused liquid staking."
-          />
-          <meta name="twitter:image" content="https://app.quicksilver.zone/img/staking-banner.png" />
-          <meta name="twitter:site" content="@QuicksilverProtocol" />
+          <meta name="twitter:description" content="STAKING SIMPLIFIED | LQUIDITY AMPLIFIED" />
```
Comment (joe-bowman) [inline web-ui/pages/about.tsx:37]: And here

**Review by joe-bowman — CHANGES_REQUESTED** [review]
Looks good except LQUIDITY

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>eb37ddf</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/Fgtx8xxXVifDVYs6uFDeQAiKZk3c'>https://vercel.com/quicksilver/quicksilver/Fgtx8xxXVifDVYs6uFDeQAiKZk3c</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9520613043)

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>085a8e7</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/6uSiH6oeTaEDQ5DQySM7tE3UAZU4'>https://vercel.com/quicksilver/quicksilver/6uSiH6oeTaEDQ5DQySM7tE3UAZU4</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9520714573)

**Review by joe-bowman — DISMISSED** [review]

**Comment by joe-bowman** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>e6619b0</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://pr-1654.app-dev.quicksilver.zone'>https://pr-1654.app-dev.quicksilver.zone</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/quicksilver/quicksilver/AF1QuTDyiyCcTYMyb4Ci4B3ixVs7'>https://vercel.com/quicksilver/quicksilver/AF1QuTDyiyCcTYMyb4Ci4B3ixVs7</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/quicksilver-zone/quicksilver/actions/runs/9521217652)

**Pull request merged**